vision/genesis_principles.md

```diff
@@ -0,0 +1,14 @@
+# ₿trust Genesis Principles
+₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust, and to act as an accountability safeguard against bad actors.  
+
+Board members will take ownership of this vision, and act in good faith to protect the interests and integrity of the organization. The specific implementation of these principles is the prerogative of the board and its selected representatives.
+
+1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork it will follow the chain endorsed by the user majority.
+
+2. ₿trust will maintain the highest levels of integrity. To that end, ₿trust does not exist to make a profit for itself or its representatives. The board will be mindful to prevent self-dealing within the organization and representatives will declare conflicts of interest as they arise.
+
+3. Free and open source projects are fundamental to Bitcoin’s censorship resistance and resilience. ₿trust will focus its efforts on fostering developer talent to work on under-resourced free and open source Bitcoin projects. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, and their capacity to accept new contributors. 
+
+4. Transparency is critical to building trust and ensuring external accountability. Decisions will be made public within a reasonable time frame and financial statements will be published quarterly. Information that undermines representatives’ security and privacy will be exempt. The ₿trust team will engage with feedback that is given in good faith to the best of its ability. 
+
+5. Decentralization of Bitcoin’s development improves its resilience, reach and relevance as a global money. ₿trust will operate in regions where Bitcoin development is uncommon.
```

Comment:
`as a global money` -> `as global money`?

Reply:
ACK on that.